extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -97,6 +104,12 @@ function SidebarWithRef(
   const [opened, setOpened] = useState(false);
   const [side, setSide] = useState(sideProp);
 
+  const [sidebarAnimation, setSidebarAnimation] = useState(null);
```

Comment:
These should be refs, not state. We don't want the effect to re-fire just because the animation has started.

Reply:
Thanks, updated in the latest.